src/dmn/trn_rpc_protocol_handlers_1.c

```diff
@@ -1257,3 +1259,56 @@ int *unload_transit_xdp_pipeline_stage_1_svc(rpc_trn_ebpf_prog_stage_t *argp,
 error:
 	return &result;
 }
+
+int *update_transit_network_policy_1_svc(rpc_trn_vsip_cidr_t *policy, struct svc_req *rqstp)
+{
+	UNUSED(rqstp);
+	static int result;
+	result = 0;
+	int rc;
+	char *itf = policy->interface;
+	struct vsip_cidr_t cidr;
+
+	TRN_LOG_INFO("update_transit_network_policy_1_svc service");
+
+	struct user_metadata_t *md = trn_itf_table_find(itf);
+	if (!md) {
+		TRN_LOG_ERROR("Cannot find interface metadata for %s", itf);
+		result = RPC_TRN_ERROR;
+		goto error;
+	}
+
+	cidr.tunnel_id = policy->tun_id;
+	cidr.prefixlen = policy->prefixlen;
+	cidr.local_ip = policy->local_ip;
+	cidr.remote_ip = policy->remote_ip;
+	__u64 bitmap = policy->bit_val;
+
+	if (policy->type == SUPPLEMENTARY) {
+		rc = trn_update_transit_supp_map(md, &cidr, bitmap);
+		if (rc != 0) {
```

Comment:
it seems trn_update_transit_supp_map(...) already have logging. Is there a reason we have another log entry here?